api/app/helpers/sendAlert.ts

```diff
@@ -1,27 +1,59 @@
 import axios from "axios";
+import { Logger } from "./logger";
 require("dotenv").config();
 
+function transformUrl(transcriptUrl?: string | null): string | null {
+  if (!transcriptUrl) {
+    Logger.error("Error transforming URL: URL not found");
+    return null;
+  }
+
+  // Extract necessary parts from the transcriptUrl
+  const { pathname } = new URL(transcriptUrl);
+  const transcriptPath = pathname.split("/").slice(5).join("/");
+
+  // Construct the correct URL using template literals
+  const transcriptUrlPath = `https://btctranscripts.com/${transcriptPath}`;
+
+  // Remove .md from the end of the URL
+  const transformedUrlPath = transcriptUrlPath.replace(/\.md$/, "");
+
+  return transformedUrlPath;
+}
+
 export async function sendAlert(
   message: string,
+  isError: boolean = false,
   transcriptTitle?: string | null,
-  transcriptUrl?: string | null,
-  transcriptHash?: string | null
+  speakers?: string | null,
```

Comment:
since this is optional no need to explicitly define the type of null
```suggestion
  speakers?: string,
```